codegen/type_converter.go

```diff
@@ -711,28 +723,61 @@ func (c *TypeConverter) genStructConverter(
 // destination fields (sent to the downstream client) and the entries are source
 // fields (from the incoming request)
 func (c *TypeConverter) GenStructConverter(
-	fromFields []*compile.FieldSpec,
-	toFields []*compile.FieldSpec,
+	fromFieldType compile.TypeSpec,
+	toFieldType compile.TypeSpec,
 	fieldMap map[string]FieldMapperEntry,
 ) error {
-	// Add compiled FieldSpecs to the FieldMapperEntry
-	fieldMap = addSpecToMap(fieldMap, fromFields, "")
-	// Check for vlaues not populated recursively by addSpecToMap
-	for k, v := range fieldMap {
-		if fieldMap[k].Field == nil {
+	helper := c.requestTypeHelper
+	var requestInput, requestOutput string
+	requestInput = "(in " + helper.FromInputType + ") " + helper.FromOutputType
+	requestOutput = "out := &" + helper.ToType + "{}\n"
+
+	c.append("func convertTo", pascalCase(helper.OutputMethodName), c.requestTypeHelper.FromSuffix,
```

Comment:
maybe `helper.ConverterMethodName` and build `pascalCase(OutputMethodName) + "ClientRequest"` in `method.go` ?